tests/phpunit/elementor/modules/home/transformations/test-filter-plugins.php

```diff
@@ -0,0 +1,106 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Modules\Home\Transformations;
+
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Modules\Home\Transformations\Filter_Plugins;
+use ElementorEditorTesting\Elementor_Test_Base;
+use PHPUnit\Framework\MockObject\MockObject;
+
+class Test_Filter_Plugins extends Elementor_Test_Base {
+
+	public function test_filter_plugins() {
```

Comment:
name should be `test_{function_name}__{scenario}`. Scenario could be optional. So here should be `test_transform`

Reply:
Every transformer will have a `transform()` method.
I think for the display conditions we have used `test_{class_name}`, which I would like to apply here as well.

Could that work?

Reply:
`test_{class_name}` is used for test class naming
`test_{function_name}` is for test function

I re-checked and for DC we used such naming as I said.

Reply:
I have left it.